custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/scala/com/datarobot/custom/H2OPredictor.scala

```diff
@@ -0,0 +1,199 @@
+package com.datarobot.custom
+
+// h2o dependencies
+import hex.genmodel.GenModel
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowData;
+import hex.genmodel.MojoModel;
+
+import scala.util.{Try, Success, Failure}
+import collection.JavaConverters._
+
+import java.util.ServiceLoader
+import java.util.ArrayList
+import java.net.URLClassLoader;
+import java.lang.Thread
+import java.io.File
+import java.io.{BufferedReader, StringReader, StringWriter}
+import java.nio.file.Paths
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+// used to compile the pojo
+import sys.process._
+
+class H2OPredictor(
+    name: String
+) extends BasePredictor(name) {
+
+  var model: EasyPredictModelWrapper = null
+  var params: java.util.HashMap[String, Any] = null
+  var isRegression: Boolean = false
+
+  var customModelPath: String = null
+  var negativeClassLabel: String = null
+  var positiveClassLabel: String = null
+
+  def predict(stringCSV: String): String = {
+
+    val csvPrinter: CSVPrinter = this.isRegression match {
+      case true =>
+        new CSVPrinter(
+          new StringWriter(),
+          CSVFormat.DEFAULT.withHeader("Predictions")
+        )
+      case false =>
+        new CSVPrinter(
+          new StringWriter(),
+          CSVFormat.DEFAULT
+            .withHeader(this.positiveClassLabel, this.negativeClassLabel)
+        )
+    }
+
+    val predictions = Try(this.scoreStringCSV(stringCSV))
+
+    predictions match {
+      case Success(preds) => {
+        isRegression match {
+          case true =>
+            preds.foreach(p =>
+              csvPrinter.printRecord(p(0).asInstanceOf[Object])
+            )
+          case false =>
+            preds.foreach { p =>
+              csvPrinter.printRecord(
+                p(1).asInstanceOf[Object],
+                p(0).asInstanceOf[Object]
+              )
+            }
+        }
+      }
+      case Failure(e) => throw new Exception(e)
```

Comment:
more exception verbiage pleas.e

Reply:
How's this? `s"During the course of making a prediction an exception was encountered: ${e}"`. here `e` will be the full exception encountered.